src/protocols/aeternity/helpers/index.ts

```diff
@@ -348,3 +350,11 @@ export function checkAddress(value: string) {
 export function checkAddressOrChannel(value: string) {
   return checkAddress(value) || isAddressValid(value, Encoding.Channel);
 }
+
+const SEED_LENGTH = 32;
+
+export function getAccountFromSecret(secretKey: Buffer) {
+  // `secretKey` variable can be either seed or seed + public key (legacy)
+  return new MemoryAccount(Buffer.from(secretKey).length === SEED_LENGTH
+    ? nacl.sign.keyPair.fromSeed(Buffer.from(secretKey)).secretKey : Buffer.from(secretKey));
```

Comment:
```suggestion
  return new MemoryAccount(
    Buffer.from(secretKey).length === SEED_LENGTH
      ? nacl.sign.keyPair.fromSeed(Buffer.from(secretKey)).secretKey
      : Buffer.from(secretKey),
  );
```
Please make it more readable by splitting the line.